colossalai/interface/optimizer.py

```diff
@@ -133,3 +133,25 @@ def unwrap(self):
         Unwrap the optimizer for checkpoint saving/loading.
         """
         return self.optim
+
+    def update_master_params(self, model: nn.Module) -> None:
```

Comment:
No need to add these methods to optimizer interface.

Reply:
modified